tutorials/irpa-customize-sales-order-creation/irpa-customize-sales-order-creation.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Customize automation generated through Recorder
+description: Customize Sales Orders Creation Bot which is generated using Recorder
+auto_validation: true
+time: 20
+tags: [ tutorial>intermediate, software-product>sap-cloud-sdk]
+primary_tag: software-product>sap-intelligent-robotic-process-automation
+---
+
+## Prerequisites
+  - [Create an automation using Recorder](irpa-win-gui-recorder) completed.
+
+## Details
+### You will learn
+  - How to customize a **Recorder** generated Automation.
+
+---
+[ACCORDION-BEGIN [Step 1: ](Customize Application)]
+Open the project where you recorded the application automation process.
+
+!![Generated automation](step2-resultant-automation.png)
+
+Select **Sales order Recorder** application.
+
+1.  Select the screen containing the order details.
+2.  Select the field containing *Material*.
+3.  Rename the material field.
+4.  Click **Remove** the **Id** **Criteria**.
+5.  Click **Name** to add it as a **Criteria**.
+6.  Click **Is a collection** !![Is a Collection icon](step2-is-a-collection-icon.PNG).
+
+    > **Is a collection** is used to declare a field as a column containing multiple fields rather than just one field.
+
+7. Repeat the steps 2-6 for the remaining two fields ( *Order Quantity Field* and *Storage location Field* ).
+
+!![Modify application fields](step2-application-modification.png)
+
+The application now recognizes *Material*, *Order Quantity* and *Storage Location* as columns and not as fields.
+
+!![Resultant Fields](step2-result.png)
+
+[VALIDATE_1]
+[ACCORDION-END]
+
+
+
+[ACCORDION-BEGIN [Step 2: ](Customize Automation)]
```

Comment:
This section's structure is not compliant with style requirements.
If a step has more than two actions, you must use a sorted list. Please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq22).